CLAUDE.md

```
@@ -0,0 +1,295 @@
+# stare — Contributor Guide
+
+Python library and CLI for the CERN ATLAS Glance/Fence API.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)